.github/renovate.json

```diff
@@ -13,6 +13,10 @@
       "matchManagers": ["gradle"],
```

Comment:
Could you please in the commit message specify which Helm chart is affected? Also a nit: Helm is written with capital "H".